tests/bluetooth/tester/prj.conf

```diff
@@ -83,7 +85,9 @@ CONFIG_BT_MESH_LIGHT_CTRL_SRV_TIME_FADE_STANDBY_MANUAL=0
 CONFIG_SETTINGS=y
 CONFIG_FLASH=y
 CONFIG_FLASH_MAP=y
-CONFIG_FCB=y
+CONFIG_NVS=y
+CONFIG_SOC_FLASH_NRF_PARTIAL_ERASE=y
+CONFIG_SOC_FLASH_NRF_PARTIAL_ERASE_MS=3
```

Comment:
this is set by default too. https://github.com/nrfconnect/sdk-zephyr/blob/main/drivers/flash/Kconfig.nrf#L60-L68
Not need